From 540cf8d11d063ea58179695a54592bafa0ce9470 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Wed, 19 Jul 2017 21:08:23 +0800
Subject: [PATCH 2167/5242] MLK-16030-1 arm64: dts: freescale: imx8mq: reduce
 power domain number

commit  903fbe03819db4e6d6ef639aff00f3ef73666a38 from
https://source.codeaurora.org/external/imx/linux-imx.git

Reduce i.MX8MQ power domain number because some power
domains can NOT support runtime ON/OFF.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index c566525..0f113cc 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -144,7 +144,7 @@
 
 	power: power-controller {
 		compatible = "fsl,imx8mq-pm-domain";
-		num-domains = <14>;
+		num-domains = <11>;
 		#power-domain-cells = <1>;
 	};
 
-- 
1.7.9.5

